spec/spec_helper.rb

```diff
@@ -110,3 +110,27 @@
   Kernel.srand config.seed
 =end
 end
+ENV["RAILS_ENV"] ||= 'test'
+require File.expand_path("../../config/environment", __FILE__)
+require 'rspec/rails'
+require 'rspec/autorun'
+
+OmniAuth.config.test_mode = true
+omniauth_hash = {
```

Comment:
This doesnt belong in the spec helper. I would create a support directory and perform this setup in a file there.

It can then be required in the rails helper